src/react-components/src/components/03-organisms/Hero/Hero.tsx

```diff
@@ -0,0 +1,115 @@
+// OH-30 Header with Image Right
+import * as React from "react";
+import bem from "../../../utils/bem";
+import Image from "../../01-atoms/Images/Image/Image";
+import Heading from "../../01-atoms/Text/Headings/Heading";
+import { HeroTypes } from "./HeroTypes";
```

Comment:
Stylistic, but I like to have these small, tightly coupled type classes in this component file rather than a separate one

Reply:
If there are more times, it might be helpful to have a util "Types.tsx" file to hold all the types. Just another option.

Reply:
I don't mind it being in a separate file so long as it's grouped within a folder. (Plus I think it keeps `Hero.tsx` from getting too long.) I'm a little more wary of a `Types.tsx` file simply because over time I think it can become a lot of code that doesn't feel as logically grouped as it would with its component.

I'm happy to move the `enum` into `Hero` if you feel strongly about it, Crystal—just want to be sure we're doing it the same way everywhere 😄

Reply:
I don't feel strongly.  Agreed that we should do it the same way everywhere.  I think button types will need to be factored out soon so we can do it the same way.

Reply:
Leaving it as-is.